rs/nns/governance/src/neuron/types.rs

```diff
@@ -811,13 +811,7 @@ impl Neuron {
             joined_community_fund_timestamp_seconds = self.joined_community_fund_timestamp_seconds;
         }
 
-        let visibility = if !is_private_neuron_enforcement_enabled() {
-            None
-        } else if self.known_neuron_data.is_some() {
-            Some(Visibility::Public as i32)
-        } else {
-            Some(self.visibility.unwrap_or(Visibility::Private) as i32)
-        };
+        let visibility = self.visibility().map(|visibility| visibility as i32);
```

Comment:
So this .visibility() function was there all along. Why wasn't it used? 

Where did the conditions is_private_neuron_enforcement_enabled() and known_neuron_data.is_some() go?

Reply:
Discussed via Zoom; fn visibility was implemented before this PR, and it should have been used here.